File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1117_basico.js
import { question } from "readline-sync";

function main() {
    let nota_1 = 0;
    let nota_2 = 0;

    while (true) {
        nota_1 = get_number();
        if(nota_1 < 0 || nota_1 > 10) {
            print(`nota invalida`);
        }else {
            break;
        }
    }

    while (true) {
        nota_2 = get_number();
        if(nota_2 < 0 || nota_2 > 10) {
            print(`nota invalida`);
        }else {
            break;
        }
    }

    let media = (nota_1 + nota_2) / 2;
    print(`media = ${media.toFixed(2)}`)
}

function get_number() {
    return parseFloat(question());
}

function print(texto) {
    return console.log(texto);
}

main();

File: Exercicios_ED_I/Bee/1117_basico.cpp
#include <stdio.h>
#include <iostream>
using namespace std;

main() {
    double n1 = 0.0;
    cin >> n1;

    while (n1 < 0.0 || n1 > 10.0) {
        cout << "nota invalida" << endl;
        cin >> n1;
    }

    double n2 = 0.0;
    cin >> n2;

    while (n2 < 0.0 || n2 > 10.0) {
        cout << "nota invalida" << endl;
        cin >> n2;
    }

    double media = (n1 + n2) / 2.0;
    printf("media = %.2lf\n", media);
}